tools/packaging/kernel/build-kernel.sh

```diff
@@ -136,6 +136,7 @@ get_tee_kernel() {
 	curl --fail -OL "${kernel_url}/linux-${kernel_tarball}" || curl --fail -OL "${kernel_url}/${kernel_tarball}"
 	
 	mkdir -p ${kernel_path}
+	tar --strip-components=1 -xf linux-${kernel_tarball} -C ${kernel_path} || \
```

Comment:
With the above, this would no longer be needed.

Reply:
done